pbrd/pbr_vty.c

```diff
@@ -557,9 +633,9 @@ DEFPY(no_pbr_map_nexthop_group, no_pbr_map_nexthop_group_cmd,
 DEFPY(pbr_map_nexthop, pbr_map_nexthop_cmd,
       "set nexthop\
         <\
-	  <A.B.C.D|X:X::X:X>$addr [INTERFACE$intf]\
-	  |INTERFACE$intf\
-	>\
+		    <A.B.C.D|X:X::X:X>$addr [INTERFACE$intf]\
```

Comment:
Please don't apply the formatting here if you don't touch the code. It's hard to read sometimes (even if it's not the logical/real change).